config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -584,6 +584,32 @@ postsubmits:
         env:
         - name: USE_BAZEL_VERSION
           value: real  # Ignore .bazelversion
+  - name: post-test-infra-push-image-benchmarkjunit
+    cluster: test-infra-trusted
+    run_if_changed: '^pkg/benchmarkjunit/'
+    annotations:
+      testgrid-dashboards: "sig-testing-images"
+      testgrid-tab-name: "benchmarkjunit"
+      testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
+      testgrid-num-failures-to-alert: '1'
+      description: builds and pushes the benchmarkerjunit image
+    decorate: true
+    branches:
+    - master
+    spec:
+      serviceAccountName: pusher
```

Comment:
I think you need the `deployer` service account like the other image publishing jobs? The comment about using `pusher` instead is saying that we should switch to using `pusher`, but that involves transferring permissions that I don't think `pusher` has yet.

Reply:
Ah gotcha, fixed